src/test-run/index.js

```diff
@@ -83,10 +103,12 @@ export default class TestRun extends Session {
         // TODO
     }
 
-    executeCommand (command) {
+    executeCommand (command, callsite) {
         if (this.pendingCommand)
             throw new Error('An attempt to execute a command when a previous command is still being executed was detected.');
 
+        this.currentCommandCallsite = callsite;
```

Comment:
> because we set pendingCommand to null when it's resolved/rejected

But why we need to store in `pendingCommand` then? We have `currentCommandCallsite` already